content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Octopus Deploy
+
+menu:
+    userguides:
+        parent: cont_delivery
+        weight: 1
+---
+
+[Octopus Deploy](https://octopus.com) is a deployment automation server, designed to make it easy to orchestrate releases and deploy applications, whether on-premises or in the cloud. Octopus is unlike a traditional CI system. In fact, Octopus is not a CI system. However, it can integrate with your existing build pipeline and integrate with several of your existing CI systems, like Jenkins, TeamCity, Azure DevOps etc.
+
+## Prerequisites
+
+- A working installation of Octopus or a hosted instance from https://octopus.com.
+- An account on the [Pulumi Console](https://app.pulumi.com).
+- The latest version of Pulumi. Installation instructions are [here]({{< relref "/docs/get-started/install" >}}).
+- Setup a new project and [stack]({{< relref "/docs/intro/concepts/stack" >}}) using one of our [Get Started]({{< relref "/docs/get-started" >}}) guides or simply by running [`pulumi new`]({{< relref "/docs/reference/cli/pulumi_new.md" >}})
+and choosing one of the many templates that are available.
+- A bare repo and set the remote URL to be your GitHub project.
```

Comment:
Perhaps "A new GitHub repository. (Or any git-based source code provider.)" instead? (Since "bare repo" and "GitHub project" are somewhat unrelated things.)